rust/opendp_tooling/src/codegen/r/mod.rs

```diff
@@ -41,7 +41,12 @@ const BLACKLIST: &'static [&'static str] = &[
     "new_queryable",
     "new_privacy_profile",
     "user_domain",
-    "_user_domain_descriptor",
+    "_make_transformation",
+    "_make_measurement",
+    "_extrinsic_domain",
+    "_extrinsic_domain_descriptor",
+    "_extrinsic_distance",
+    "_extrinsic_divergence",
```

Comment:
Not sure how to be confident that this is up to date. If there is something that's already checking this, add a note at the top?

Reply:
If these are missing, the R package will generate code that attempts to bind to these functions, and that generated code will raise errors/cause failures.